stable/rabbitmq-ha/templates/statefulset.yaml

```diff
@@ -155,11 +155,10 @@ spec:
               mountPath: /var/lib/rabbitmq
             - name: config
               mountPath: /etc/rabbitmq
-            {{- if not .Values.existingSecret }}
+              readOnly: true
```

Comment:
Remove this as it was just removed in PR #9222

Reply:
Done